pkg/pillar/Makefile

```diff
@@ -81,6 +81,8 @@ build-docker-git:
 	git archive HEAD | docker build $(DOCKER_ARGS) -t $(DOCKER_TAG) -
 
 build-docker-test:
+	../../build-tools/bin/linuxkit cache export $(shell ../../build-tools/bin/linuxkit pkg show-tag ../alpine) --outfile - --format docker | docker image load
+	../../build-tools/bin/linuxkit cache export $(shell ../../build-tools/bin/linuxkit pkg show-tag ../dom0-ztools/) --outfile - --format docker | docker image load
```

Comment:
Why are you exporting only this packages? Also, you can just use: `$(MAKE) -C ../../ alpine-cache-export-docker-load dom0-ztools-cache-export-docker-load`